.github/workflows/ci.yml

```diff
@@ -0,0 +1,112 @@
+name: CI
+on:
+  push:
+    branches:
+      - "master"
+  pull_request:
+    branches:
+      - "**"
+    types: [opened, synchronize]
+
+concurrency:
+  group: build-${{ github.event.pull_request.number || github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  # TODO adapt the whole integration tests to use https://www.notion.so/polygontechnology/No-More-Secrets-6382754faac745aa9bd049a3ed423218
+  integration-tests:
+    if: (github.event.action != 'closed' || github.event.pull_request.merged == true)
+    strategy:
+      matrix:
+        os: [ ubuntu-20.04 ] # list of os: https://github.com/actions/virtual-environments
+    runs-on: ${{ matrix.os }}
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          path: matic-cli
+
+      - name: Create .env file
+        continue-on-error: true
+        run: |
+          cd matic-cli
+          cp .env.example .env
+          sed -i 's,YOUR_IDENTIFIER,matic-cli-ci,' .env
+          sed -i 's,YOUR_AWS_ACCESS_KEY,TODO,' .env
+          sed -i 's,YOUR_AWS_SECRET_KEY,TODO,' .env
+          sed -i 's,aws-key,matic-cli-ci-key,' .env
+          sed -i 's,/absolute/path/to/your/,/home/runner/work/matic-cli/matic-cli/matic-cli/aws/,' .env
+          sed -i 's,MATIC_CLI_BRANCH=master,MATIC_CLI_BRANCH=${{ github.head_ref || github.ref_name }},' .env
+          sed -i 's,DATADOG_API_KEY,TODO,' .env
+          cat .env
+
+      - name: Setup aws
+        continue-on-error: true
+        run: |
+          cd matic-cli
```

Comment:
one day we'd need to make the same with ansible

Reply:
Yes :)